InfiniteRecharge/src/test/java/frc/robot/subsystem/drive/auto/FieldConstantsTest.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystem.drive.auto;
+
+import java.util.List;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import frc.robot.subsystem.drive.auto.FieldConstants;
+
+
+
+public class FieldConstantsTest {
+    @Test
+    public void testRelativeTransform() {
+        Translation2d pos = FieldConstants.transformToRobot(FieldConstants.OUR_POWER_CELL_1);
+
+        assertEquals(pos.getX(), 3.114802, 0.00001);
```

Comment:
Shouldn’t this be (1.5, 1.2)? That’s what worked in our tests and you are using this transformed position in the trajectory now.